src/Illuminate/Database/Eloquent/Factory.php

```diff
@@ -5,53 +5,32 @@
 use ArrayAccess;
 use Faker\Generator as Faker;
 use Symfony\Component\Finder\Finder;
+use Illuminate\Database\Eloquent\Factory\StateManager;
 
 class Factory implements ArrayAccess
 {
-    /**
-     * The model definitions in the container.
-     *
-     * @var array
-     */
-    protected $definitions = [];
-
-    /**
-     * The registered model states.
-     *
-     * @var array
-     */
-    protected $states = [];
-
-    /**
-     * The registered after making callbacks.
-     *
-     * @var array
-     */
-    protected $afterMaking = [];
-
-    /**
-     * The registered after creating callbacks.
-     *
-     * @var array
-     */
-    protected $afterCreating = [];
-
     /**
      * The Faker instance for the builder.
      *
      * @var \Faker\Generator
      */
     protected $faker;
 
+    /**
+     * @var \Illuminate\Database\Eloquent\Factory\StateManager
```

Comment:
Add a small one line explanation of this property like on other properties/classes.